### ISSUE TITLE
Ensure the parser runs properly on major PHP frameworks/libraries

### ISSUE THREAD

**Issue by nevadascout** [HvyIndustries/crane#148]
- [ ] Symfony
- [ ] Laravel
- [ ] CodeIgniter
- [ ] Yii 2
- [ ] Yii 1
- [ ] Nette
- [ ] Phalcon
- [ ] CakePHP
- [ ] Zend Framework
- [ ] Slim
- [ ] Fuel PHP
- [ ] PHPixie

**Comment by nevadascout** [comment]
These frameworks all need retesting once the issues marked against them are resolved

**Comment by nevadascout** [comment]
Deferring to 0.2.2 release

**Comment by Novynn** [comment]
Could we also add Zend Framework 1 to the list? 1.12.20 is the latest. Still failing to finishing parsing with it included as of Crane v0.2.1.

**Comment by nevadascout** [comment]
I've added Zend Framework 1, thanks for the suggestion!

**Comment by mpotoc** [comment]
Is it possible to make Phalcon also work? I tried and tried with Phalcon stubs to get them working, but no luck so far. Am i wrong or not possible to do that yet?

**Comment by ichiriac** [comment]
@mpotoc you mean you would like to parse files like these one : https://github.com/phalcon/cphalcon/blob/master/phalcon/acl.zep ?

**Comment by mpotoc** [comment]
@ichiriac yes something like that, i am using phpstorm (university license until mid 2017) and in there you can add external libraries like Phalcon Dev Tools, which have stubs folder under IDE folder and you can link this in phpstorm to get full Phalcon namespace, etc. intellisense. Is something like this possible in VS Code with your extension, or will be anytime soon? Still have 8 month to decide stay on Storm and buy it or start using something else.

Thanks

**Comment by nevadascout** [comment]
@mpotoc Yes, we are planning to add a feature that lets you get intellisense for libraries that are outside the workspace folder. This change will also let you define filetypes other than `.php` that the parser should parse.

**Comment by mpotoc** [comment]
@ichiriac perfect, will look for updates of your extension. Thanks a lot for the answer.

**Comment by kiatng** [comment]
Is it possible to add Magento 1.9.x and 2.x to the list? For Magento 1.9.x, it stops at _66 of 9748 / 0.7%_

**Comment by ichiriac** [comment]
@kiatng they are now in the php-parser token tests en ensure their parsing (bugfixing in progress)

**Comment by scottsb** [comment]
@ichiriac would that apply to Magento 2 as well, do you think? It also locks up having partially indexed.

**Comment by ichiriac** [comment]
@scottsb already done, here the list of projects already passing : 
https://github.com/glayzzle/php-parser-dev/tree/master/test/token

**Comment by codewp** [comment]
Is it possible to adding Wordpress to this list?
Indexing stuck at 48%

**Comment by ichiriac** [comment]
hi @codewp, it's done & tests passes with the latest version - now we need to [release crane](https://github.com/HvyIndustries/crane/issues/183) in order to fix all these bugs

**Comment by nevadascout** [comment]
All major libraries in this list now parse without any errors
